exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/statistics/StatisticsMergeBatch.java

```diff
@@ -82,17 +83,27 @@
  *       "sales_city" : BIGINT - nonnullstatcount(sales_city)
  *       "cnt"        : BIGINT - nonnullstatcount(cnt)
  *   .... another map for next stats function ....
+ * </pre>
+ * <p>
+ * Note that the above schema is not a valid Drill record batch: the varous
```

Comment:
```suggestion
 * Note that the above schema is not a valid Drill record batch: the various
```

Reply:
@vvysotskyi could you please take a look at this java doc comment and changes in this class?

Reply:
If I understand these changes from DRILL-1328 correctly, as a result, we should obtain a map with a single value and its children also should have single values. Looks like this is a bug with specifying wrong row count (perhaps in `buildOutgoingRecordBatch()` method).